content/rs/release-notes/rs-6-2-4-august-2021.md

```diff
@@ -0,0 +1,159 @@
+
+    Setting the upgrade policy to `latest` ensures that the most recent Redis features are available to new databases and ones that are upgraded.  It also requires more frequent upgrades, as open source Redis is updated more frequently than Redis Enterprise Software.
+
+The Redis Enterprise Software 6.2.4 package includes compatibility with the most recent major Redis release (v6.0) and the latest (most recent) update to Redis (v6.2.3).  
+
+By default, compatibility with v6.0 will be installed.  To change this, use `rladmin` to set the upgrade policy and the default Redis version:
+
+``` shell
+tune cluster redis_upgrade_policy latest
+tune cluster default_redis_version 6.2
+```
+
+To learn more, see the [upgrade instructions]({{<relref "/rs/installing-upgrading/upgrading.md">}}).
+
+### Product lifecycle updates
+
+Redis Enterprise Software v5.6.0 will reach end of life (EOF) on October 31, 2021.
+
+To learn more, see the Redis Enterprise Software [product lifecycle]({{<relref "/rs/administering/product-lifecycle.md">}}), which details the release number and the end-of-life schedule for Redis Enterprise Software.
+
+Redis Enterprise modules have individual release numbers [and lifecycle]({{<relref "/modules/modules-lifecycle.md">}}).
+
+### Deprecation notices
+
+- [In v6.0.20]({{<relref "/rs/release-notes/rs-6-0-20-april-2021.md">}}), the SASL-based LDAP mechanism was deprecated in favor of a new [RBAC-based approach]({{<relref "/rs/security/ldap/">}}).  As of v.6.2.4, support for the older mechanism has been removed.
+
+    For help migrating to the LDAP-based mechanism, see [Migrate to role-based LDAP]({{<relref "/rs/security/ldap/migrate-to-role-based-ldap.md">}}).
+    
+- [OpenStack Object Storage](https://wiki.openstack.org/wiki/Swift) ("Swift") has reached end-of-life.  Consequently, you can no longer use ObjectStack Swift as a target for database backup or export operations.  
+
+## Features and enhancements
+
+### Internode encryption
+
+Internode encryption (INE) encrypts all communication between nodes in a cluster; it is available for the control plan and the data plane. 
+
+#### Control plane internode encryption
+
+Control plane internode encryption encrypts all management communication with a cluster.  it is enabled by default for all new clusters and upgraded clusters.
+
+#### Data plane internode encryption
+
+Data plane internode encryption encrypts communication between nodes within a cluster, such as database replication between nodes.  
+
+Data plane internode encryption is available for new or fully upgraded clusters.  It not enabled by default.  
+
+You can enable data plane internode encryption by:
+
+- Setting the cluster policy to enable data plane internode encryption by default for new databases
+    
+    ``` shell
+    rladmin tune cluster data_internode_encryption
```

Comment:
Just checking since I can't quite remember, is it true that this command doesn't need to specify "data_internode_encryption enabled"?

Reply:
I'll check.